### DISCUSSION TITLE
Cant Authorize Github Desktop

### DISCUSSION THREAD

**Discussion by TherealONE12** [community/community#114460] [New to GitHub]
I have fully verified my Account, but the button for verifying wount be clickable. 
![image](https://github.com/community/community/assets/144170391/30fb51c9-3ba1-4170-b07b-2f15ce5594ef)
Does it have something to do with this marked thingy? 
![image](https://github.com/community/community/assets/144170391/7a1cbf37-b278-4c3c-8fcd-5ce5f6874da0)
 
 
 Any help is greatly appreciated. Thank you in advance!

**Comment by habibthadev** [comment]
The image shows a screen for authorizing access to repositories, personal user data, and workflow. If the "Authorize desktop" button is not clickable, it may be due to a technical issue or glitch with the website or application. In such cases, the recommended steps would be to refresh the page, clear the browser cache and cookies, or try accessing the site from a different device or network connection. If the issue persists, contacting the support team for further assistance would be advisable.

**Reply by TherealONE12** [reply]
yeah its still ther, but i cant contact support as it is only for paid people. (as far as my knowledge goes)
